docs/get-started/getting-started-alluxio.md

```diff
@@ -365,3 +365,49 @@ This section will give some links about how to configure, tune Alluxio and some
 - [Running Spark on Alluxio](https://docs.alluxio.io/os/user/stable/en/compute/Spark.html)
 - [Performance Tuning](https://docs.alluxio.io/os/user/stable/en/operation/Performance-Tuning.html )
 - [Alluxio troubleshooting](https://docs.alluxio.io/os/user/stable/en/operation/Troubleshooting.html)
+
+## Alluxio reliability
+The properties mentioned in this section can be found in [Alluxio configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)
+
+### Avoid long time response if masters or workers hang
+The default value of `alluxio.user.rpc.retry.max.duration` is 2 minutes.  
+If master is done or hang, the `alluxio.user.rpc.retry.max.duration` will cause at least 2 minutes waiting, it's too long.
+The default value of `alluxio.user.block.read.retry.max.duration` is 5 minutes.
+If worker is done or hang, the `alluxio.user.block.read.retry.max.duration` will cause 5 minutes waiting, it's too long.
+
+See also:
+```
+alluxio.user.rpc.retry.max.duration
+alluxio.user.rpc.retry.max.sleep	
+alluxio.user.rpc.retry.base.sleep	
+
+alluxio.user.block.read.retry.max.duration
+alluxio.user.block.read.retry.sleep.max		
+alluxio.user.block.read.retry.sleep.base 		
+```
+Above configurations define the `ExponentialTimeBoundedRetry` retry policies and `max durations`, we can adjust them to appropriate values.  
+
+Set these properties on Spark because Spark invokes Alluxio client.
+vi $SPARK_HOME/conf/spark-defaults.conf
+```
+spark.driver.extraJavaOptions    -Dalluxio.user.rpc.retry.max.duration=10sec -Dalluxio.user.block.read.retry.max.duration=10sec
+spark.executor.extraJavaOptions  -Dalluxio.user.rpc.retry.max.duration=10sec -Dalluxio.user.block.read.retry.max.duration=10sec
+```
+
+### Worker Timeout to fail fast
```

Comment:
so this header the last header seem very similar, I think perhaps the difference is client vs server configuration, correct?  If so lets add something about that to the title

Reply:
Perhaps like "Worker server tunings to fail fast"